### PR TITLE
Update debug.md

### PR DESCRIPTION
**Pull request by chenxinlong**
wrong kubevirt cr example

### CODE REVIEW

File: docs/operations/debug.md [hunk 1]
```diff
@@ -49,9 +49,10 @@ metadata:
 spec:
   configuration:
     developerConfiguration:
-      nodeVerbosity:
-        "node01": 4
-        "otherNodeName": 6
+      logVerbosity
```
Comment (jean-edouard) [inline docs/operations/debug.md:52]:
```suggestion
      logVerbosity:
```
?

**Comment by cwilkers** [comment]
Thanks for the PR!

We could use a reviewer who is familiar with the API in question. Do you have somebody in mind? (you can add them with /cc <at>githubuser)

/ok-to-test

**Comment by aburdenthehand** [comment]
This looks correct as per [the v1.0 API reference](https://kubevirt.io/api-reference/v1.0.0/definitions.html#_v1_developerconfiguration)
@jean-edouard can you confirm?
@chenxinlong can you please rebase and resolve merge conflict?

**Pull request closed without merging**